crates/npmrc/src/custom_deserializer.rs

```diff
@@ -16,6 +16,18 @@ pub fn default_public_hoist_pattern() -> Vec<String> {
     vec!["*eslint*".to_string(), "*prettier*".to_string()]
 }
 
+pub fn is_home(path: String) -> bool {
+    path.starts_with("~/") || path.starts_with("~\\")
+}
```

Comment:
Normally, I would suggest you to use `&str` in place of `String` to avoid `.clone()`, but there's a better solution.

Please keep this in mind anyway. There are also other tips in our [CODE_STYLE_GUIDE](https://github.com/pnpm/pacquet/blob/main/CODE_STYLE_GUIDE.md).

Reply:
Actually, rename this to `strip_home_prefix`:

```rust
fn strip_home_prefix(path: &str) -> Option<&'_ str> {
    path.strip_prefix("~/").or_else(|| path.strip_prefix("~\\"))
}
```